revoke/revoke.go

```diff
@@ -142,6 +158,31 @@ func certIsRevokedCRL(cert *x509.Certificate, url string) (revoked, ok bool) {
 		}
 	}
 
+	u, err := neturl.Parse(url)
+	if err != nil {
+		log.Warningf("failed to parse CRL url: %v", err)
+		return false, false
+	}
+	if u.Scheme == "" && shouldFetchCRL {
```

Comment:
@lziest RFC explains the certificate extension, and there is nothing about the local file. In our situation we set CRL path from the app and it doesn't relate to this RFC.